shell/platform/android/io/flutter/embedding/engine/android/FlutterView.java

```diff
@@ -300,6 +301,27 @@ public boolean onHoverEvent(MotionEvent event) {
   }
   //-------- End: Process UI I/O that Flutter cares about. ---------
 
+  //-------- Start: Accessibility -------
+  public void updateAccessibilityFeatures() {
+    // TODO(mattcarroll): get the following accessibility code working. This method is added now to
+    //                    avoid forgetting its invocation in FlutterFragment, but I want to think
+    //                    more deeply about how much accessibility logic should truly live in
+    //                    FlutterView vs AccessibilityBridge.
+//    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+//      String transitionAnimationScale = Settings.Global.getString(getContext().getContentResolver(),
+//          Settings.Global.TRANSITION_ANIMATION_SCALE);
+//      if (transitionAnimationScale != null && transitionAnimationScale.equals("0")) {
+//        accessibilityFeatureFlags ^= AccessibilityFeature.DISABLE_ANIMATIONS.value;
```

Comment:
This should not be `^=`, it should be `|=`.  There was a bug in some related code that I fixed for this, but looks like I had missed it here.

Reply:
Done